psychopy/layout.py

```diff
@@ -167,16 +269,20 @@ def __mul__(self, other):
             return Vector(self.pix * other.pix, "pix", self.win)
         if isinstance(other, (int, float)):
             return Vector(self.pix * other, "pix", self.win)
-        if isinstance(other, (list, tuple, numpy.ndarray)):
-            return Vector(self.pix * numpy.array(other), "pix", self.win)
+        if isinstance(other, (list, tuple, np.ndarray)):
+            return Vector(self.pix * np.array(other), "pix", self.win)
 
     def __truediv__(self, other):
         if isinstance(other, Vector):
             return Vector(self.pix / other.pix, "pix", self.win)
         if isinstance(other, (int, float)):
             return Vector(self.pix / other, "pix", self.win)
-        if isinstance(other, (list, tuple, numpy.ndarray)):
-            return Vector(self.pix / numpy.array(other), "pix", self.win)
+        if isinstance(other, (list, tuple, np.ndarray)):
+            return Vector(self.pix / np.array(other), "pix", self.win)
+
+    # --------------------------------------------------------------------------
+    # Class methods and properties
```

Comment:
This is a good shout, I'll start doing this in other classes, really tidies things up